pom.xml

```diff
@@ -549,18 +578,30 @@
         <scope>compile</scope>
         <exclusions>
           <exclusion>
-            <artifactId>commons-digester</artifactId>
-            <groupId>commons-digester</groupId>
+            <artifactId>commons-digester3</artifactId>
+            <groupId>commons-digester3</groupId>
```

Comment:
no sure commons-configuration have both dependencies digester and digester3, so most probably you do not need to change this exclusion